src/client/datascience/insidersNativeNotebookSurveyBanner.ts

```diff
@@ -127,8 +127,8 @@ export class InsidersNativeNotebooksSurveyBanner implements IExtensionSingleActi
         this.browserService.launch('https://aka.ms/vscjupyternb');
     }
 
-    private async isInsidersNativeNotebooksUser() {
-        return this.useVSCodeNotebookEditorApi;
+    private isInsidersNativeNotebooksUser() {
```

Comment:
Can we rename this file and method to `nativeNotebooksSurveyBanner` and `nativeNotebooksUser` respectively?

Reply:
Nope, this banner is still applicable only for VS Code Insiders, & nothing else.

Reply:
I.e file name & class name is still correct.